### PR TITLE
add cnv-odf-tests-aws-ipi-ocp418-fips

### PR DESCRIPTION
**Pull request by amp-rh**
Rebase of #56412
Closes [INTEROP-7510](https://issues.redhat.com/browse/INTEROP-7510) 

Co-authored-by: Diane Frazzette 
 <dfrazzet@redhat.com>

Co-authored-by: oharan2
  <oharan@redhat.com>

### CODE REVIEW

**Comment by amp-rh** [comment]
/pj-rehearse pull-ci-RedHatQE-interop-testing-cnv-4.18-cnv-odf-ocp4.18-lp-interop-images

File: ci-operator/config/RedHatQE/interop-testing/RedHatQE-interop-testing-cnv-4.18__cnv-odf-ocp4.18-lp-interop.yaml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+              {"step": "*ocs-tests"            , "failure_type": "test_failure" , "classification": "ODF Storage Tests Failure"             , "group": { "name" : "lp-tests", "priority" : 1}, "jira_additional_labels": ["!default", "interop-tests"], "jira_project": "RHSTOR"},
+              {"step": "*cnv-tests-e2e-deploy" , "failure_type": "pod_failure"  , "classification": "CNV Tests Execution"                   , "group": { "name" : "lp-tests", "priority" : 2}, "jira_additional_labels": ["!default", "interop-tests"]                          },
+              {"step": "*cnv-tests-e2e-deploy" , "failure_type": "test_failure" , "classification": "CNV Tests Failure"                     , "group": { "name" : "lp-tests", "priority" : 2}, "jira_additional_labels": ["!default", "interop-tests"], "jira_project": "CNV"   }
+            ]
+        }
+      FIREWATCH_CONFIG_FILE_PATH: https://raw.githubusercontent.com/CSPI-QE/cspi-utils/main/firewatch-base-configs/aws-ipi/lp-interop.json
+      FIREWATCH_DEFAULT_JIRA_ADDITIONAL_LABELS: '["4.18-lp", "self-managed-lp", "cnv-lp",
+        fips]'
```
Comment (dfrazzette) [inline ci-operator/config/RedHatQE/interop-testing/RedHatQE-interop-testing-cnv-4.18__cnv-odf-ocp4.18-lp-interop.yaml:62]:
```suggestion
      FIREWATCH_DEFAULT_JIRA_ADDITIONAL_LABELS: '["4.18-lp", "self-managed-lp", "cnv-lp","fips"]'
```

File: ci-operator/config/RedHatQE/interop-testing/RedHatQE-interop-testing-cnv-4.18__cnv-odf-ocp4.18-lp-interop.yaml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+      product: ocp
+      stream: nightly
+      version: "4.18"
+resources:
+  '*':
+    requests:
+      cpu: 200m
+      memory: 400Mi
+tests:
+- as: cnv-odf-tests-aws-ipi-ocp418-fips
```
Comment (dfrazzette) [inline ci-operator/config/RedHatQE/interop-testing/RedHatQE-interop-testing-cnv-4.18__cnv-odf-ocp4.18-lp-interop.yaml:40]:
```suggestion
- as: cnv-odf-tests-aws-ipi-ocp418
  cron: 0 23 31 2 *
  steps:
    cluster_profile: aws-cspi-qe
    env:
      BASE_DOMAIN: cspilp.interop.ccitredhat.com
      COMPUTE_NODE_TYPE: c5n.metal
      FIREWATCH_CONFIG: |
        {
          "failure_rules":
            [
              {"step": "cucushift-*"           , "failure_type": "all"          , "classification": "Cluster Health Check"                  , "group": { "name" : "cluster" , "priority" : 2}, "jira_additional_labels": ["!default"]                                           },
              {"step": "*deploy-odf"           , "failure_type": "all"          , "classification": "Operator Setup"                        , "group": { "name" : "lp-tests", "priority" : 1}, "jira_additional_labels": ["!default", "interop-tests"]                          },
              {"step": "*ocs-tests"            , "failure_type": "pod_failure"  , "classification": "ODF Storage Tests Execution"           , "group": { "name" : "lp-tests", "priority" : 1}, "jira_additional_labels": ["!default", "interop-tests"]                          },
              {"step": "*ocs-tests"            , "failure_type": "test_failure" , "classification": "ODF Storage Tests Failure"             , "group": { "name" : "lp-tests", "priority" : 1}, "jira_additional_labels": ["!default", "interop-tests"], "jira_project": "RHSTOR"},
              {"step": "*cnv-tests-e2e-deploy" , "failure_type": "pod_failure"  , "classification": "CNV Tests Execution"                   , "group": { "name" : "lp-tests", "priority" : 2}, "jira_additional_labels": ["!default", "interop-tests"]                          },
              {"step": "*cnv-tests-e2e-deploy" , "failure_type": "test_failure" , "classification": "CNV Tests Failure"                     , "group": { "name" : "lp-tests", "priority" : 2}, "jira_additional_labels": ["!default", "interop-tests"], "jira_project": "CNV"   }
            ]
        }
      FIREWATCH_CONFIG_FILE_PATH: https://raw.githubusercontent.com/CSPI-QE/cspi-utils/main/firewatch-base-configs/aws-ipi/lp-interop.json
      FIREWATCH_DEFAULT_JIRA_ADDITIONAL_LABELS: '["4.18-lp", "self-managed-lp", "cnv-lp"]'
      FIREWATCH_DEFAULT_JIRA_PROJECT: LPINTEROP
      FIREWATCH_FAIL_WITH_TEST_FAILURES: "true"
      OCP_VERSION: "4.18"
      ODF_OPERATOR_CHANNEL: stable-4.18
      ODF_VERSION_MAJOR_MINOR: "4.18"
      RE_TRIGGER_ON_FAILURE: "false"
      USER_TAGS: |
        scenario cnv
    test:
    - chain: cucushift-installer-check-cluster-health
    - ref: interop-tests-deploy-odf
    - ref: interop-tests-ocs-tests
    - ref: interop-tests-cnv-tests-e2e-deploy
    - ref: interop-tests-openshift-virtualization-tests
    workflow: firewatch-ipi-aws
- as: cnv-odf-tests-aws-ipi-ocp418-fips
```

**Comment by dfrazzette** [comment]
Although the rehearsal jobs have failed, we plan to investigate and debug as part of the weekly OCP 4.18 weekly runs once PR is merged.

**Review by rabin-io — APPROVED** [review]
/approved

**Pull request merged**